app/api/__init__.py

```diff
@@ -359,3 +361,12 @@
 api.route(CustomFormDetail, 'custom_form_detail', '/custom-forms/<int:id>')
 api.route(CustomFormRelationshipRequired, 'custom_form_event',
           '/custom-forms/<int:id>/relationships/event')
+
+# Stripe Authorization API
+api.route(StripeAuthorizationListPost, 'stripe_authorization_list_post', '/stripe-authorization')
+api.route(StripeAuthorizationList, 'stripe_authorization_list', '/events/<int:event_id>/stripe',
+          '/events/<event_identifier>/stripe')
+api.route(StripeAuthorizationDetail, 'stripe_authorization_detail',  '/stripe-authorization/<int:id>',
```

Comment:
@niranjan94 as you told, I changed it to "/events/<int:event_id>/stripe-authorization"